nfs/README.md

```diff
@@ -0,0 +1,121 @@
+# Using an NFS to allow a pod to access local Files
+
+## Creating the Persistent Volume
+
+First, find a persistent volume in the right region
+
+| Host Name     | Region | Location           | IP |
+|----------|--------|--------------|--------------------|
+| banff-1e707-e02-2.mkm.dcgpu | CA-GTA | Markham Campus  | rmm-banff-1e707-e02.amd.com |
+| smc300x-clt-r4c4-34.cs-clt.dcgpu | US-Southeast | Charlotte Cirrascale Colo | 10.235.86.34 |
+| smc300x-clt-r4c6-26.cs-clt.dcgpu | US-Southeast | Charlotte Cirrascale Colo | 10.235.86.43 |
+| smc300x-clt-r4c6-34.cs-clt.dcgpu | US-Southeast | Charlotte Cirrascale Colo | 10.235.86.44 |
+| banff-1e707-f07-5.mkm.dcgpu | CA-GTA | Markham Campus | rmm-banff-1e707-f07.amd.com |
+| banff-sc-cs47-05.dh170.dcgpu | US-BayArea | Santa Clara DH170 Lab | 10.216.110.62 |
+| dell300x-ccs-aus-B17-19.cs-aus.dcgpu | US-Texas | Austin Cirrascale Colo | 10.235.28.121 |
+| SMC-SC-DI09-03.dh144.dcgpu | US-BayArea | Santa Clara DH144 Lab | 10.216.113.229 |
+
+you can look at available persistent volumes by running `kubectl get pv`
+
+If you don't have a persistent volume, you can request one by filing a ticket here https://confluence.amd.com/display/SHARK/OSSCI+Customer+Instructions
```

Comment:
@yamiyysu I addressed the comment here. I can give more detailed directions though